src/main/java/sirius/kernel/health/console/LoggerCommand.java

```diff
@@ -33,7 +33,8 @@ public void execute(Output output, String... params) {
             output.line("Known loggers:");
             output.separator();
             for (Log l : Log.getAllLoggers()) {
-                output.apply("%-30s %-10s", l.getName(), l.getLevel());
+                Level logLevel = l.getLevel() != null ? l.getLevel() : Level.INFO;
```

Comment:
shouldn't we rather fix this in Log.getLevel()? kind of strange to return null there(?)

Reply:
Hab dir schon dazu geschrieben :D Eigentlich ists scheinbar von Java so vorgegeben dass das Loglevel null sein kann